buildpack/databroker/__init__.py

```diff
@@ -0,0 +1,153 @@
+#
+# [EXPERIMENTAL]
+#
+# Add databroker components to an mx app container
+#
+
+import os
+import logging
+import json
+
+from buildpack.databroker import connect, streams
+from buildpack.databroker.config_generator.scripts.configloader import (
+    configinitializer,
+)
+
+DATABROKER_ENABLED_FLAG = "DATABROKER_ENABLED"
+RUNTIME_DATABROKER_FLAG = "DATABROKER.ENABLED"
+
+APP_MODEL_HOME = "/home/vcap/app/model"
+METADATA_FILE = os.path.join(APP_MODEL_HOME, "metadata.json")
+DEP_FILE = os.path.join(APP_MODEL_HOME, "dependencies.json")
+
+MAX_DATABROKER_COMPONENT_RESTART_RETRIES = 4
+
+
+def is_enabled():
+    if os.environ.get(DATABROKER_ENABLED_FLAG) == "true":
+        logging.debug("Databroker is enabled")
+        return True
+    else:
+        return False
+
+
+def is_producer_app():
+    if not is_enabled():
+        return False
+
+    with open(METADATA_FILE) as f:
+        metadata_json = json.load(f)
+
+    db_config = metadata_json.get("DataBrokerConfiguration")
+    if (
+        db_config != None
+        and db_config.get("publishedServices") != None
+        and len(db_config.get("publishedServices")) > 0
+    ):
+        return True
+    else:
+        return False
+
+
+def should_run_kafka_connect():
+    try:
+        if (
+            os.environ["CF_INSTANCE_INDEX"] != None
+            and int(os.environ["CF_INSTANCE_INDEX"]) == 0
+        ):
+            return True
+        else:
+            return False
```

Comment:
```python
return (os.environ["CF_INSTANCE_INDEX"] != None and int(os.environ["CF_INSTANCE_INDEX"]) == 0)
```
Nitpicking. It can be expressed more concisely, if desired. I'm typically not a big fan of explicitly returning `True` (and `False` in the else) statement.

Reply:
My suggestion turns it into a one liner ;)

Reply:
@abhishekamte-mx - can you respond?

Reply:
sure i can make the change, its a good suggestion to make the code more "pythonic". any other comments @svanderburg  @markbastiaans ?

Reply:
i have incorporated the change